include/TrustWalletCore/TWP2PKHPrefix.h

```diff
@@ -17,7 +17,7 @@ enum TWP2PKHPrefix {
     TWP2PKHPrefixBitcoin = 0x00,
     TWP2PKHPrefixDash = 0x4C,
     TWP2PKHPrefixDecred = 0x3f,
-    TWP2PKHPrefixDogecoin = 0x1e,
+    TWP2PKHPrefixDogecoin_DigiByte = 0x1e,
```

Comment:
or maybe just `TWP2PKHPrefixD`? and add comment // for Doge/DigiByte etc

Reply:
Done in 84bc3aa